### ISSUE TITLE
Unknown WKB type 8

### ISSUE THREAD

**Issue by m-kuhn** [claeis/iox-ili#92]
Exporting a dataset with MultiSurface from postgis, ili2pg errors out when a curvepolygon contains a circularstring directly (not wrapped in a compoundcurve)

```
Error: line 0: Administrative_Einteilung_LV95_V1_0.Administration.Einteilung: tid 731: Attribute Geometrie has wrong number of values
Error: Object 732: failed to convert surface/area
Error:   com.vividsolutions.jts.io.ParseException: Unknown WKB type 8
Error:       ch.ehi.ili2pg.converter.PostgisColumnConverter.toIomMultiSurface(PostgisColumnConverter.java:450)
Error:       ch.ehi.ili2db.toxtf.ToXtfRecordConverter.addAttrValue(ToXtfRecordConverter.java:629)
Error:       ch.ehi.ili2db.toxtf.ToXtfRecordConverter.convertRecord(ToXtfRecordConverter.java:441)
Error:       ch.ehi.ili2db.toxtf.TransferToXtf.dumpObjHelper(TransferToXtf.java:1104)
Error:       ch.ehi.ili2db.toxtf.TransferToXtf.dumpObject(TransferToXtf.java:1080)
Error:       ch.ehi.ili2db.toxtf.TransferToXtf.doBasket(TransferToXtf.java:564)
Error:       ch.ehi.ili2db.toxtf.TransferToXtf.doit(TransferToXtf.java:293)
Error:       ch.ehi.ili2db.base.Ili2db.transferToXtf(Ili2db.java:2300)
Error:       ch.ehi.ili2db.base.Ili2db.runExport(Ili2db.java:1720)
Error:       ch.ehi.ili2db.base.Ili2db.run(Ili2db.java:225)
Error:       ch.ehi.ili2db.AbstractMain.domain(AbstractMain.java:554)
Error:       ch.ehi.ili2pg.PgMain.main(PgMain.java:70)
Error:     Unknown WKB type 8
Error:         ch.interlis.iox_j.wkb.Wkb2iox.readGeometry(Wkb2iox.java:187)
Error:         ch.interlis.iox_j.wkb.Wkb2iox.readCurvePolygon(Wkb2iox.java:386)
Error:         ch.interlis.iox_j.wkb.Wkb2iox.readMultiSurface(Wkb2iox.java:469)
Error:         ch.interlis.iox_j.wkb.Wkb2iox.readGeometry(Wkb2iox.java:185)
Error:         ch.interlis.iox_j.wkb.Wkb2iox.read(Wkb2iox.java:131)
Error:         ch.interlis.iox_j.wkb.Wkb2iox.read(Wkb2iox.java:112)
Error:         ch.ehi.ili2pg.converter.PostgisColumnConverter.toIomMultiSurface(PostgisColumnConverter.java:448)
Error:         ch.ehi.ili2db.toxtf.ToXtfRecordConverter.addAttrValue(ToXtfRecordConverter.java:629)
Error:         ch.ehi.ili2db.toxtf.ToXtfRecordConverter.convertRecord(ToXtfRecordConverter.java:441)
Error:         ch.ehi.ili2db.toxtf.TransferToXtf.dumpObjHelper(TransferToXtf.java:1104)
Error:         ch.ehi.ili2db.toxtf.TransferToXtf.dumpObject(TransferToXtf.java:1080)
Error:         ch.ehi.ili2db.toxtf.TransferToXtf.doBasket(TransferToXtf.java:564)
Error:         ch.ehi.ili2db.toxtf.TransferToXtf.doit(TransferToXtf.java:293)
Error:         ch.ehi.ili2db.base.Ili2db.transferToXtf(Ili2db.java:2300)
Error:         ch.ehi.ili2db.base.Ili2db.runExport(Ili2db.java:1720)
Error:         ch.ehi.ili2db.base.Ili2db.run(Ili2db.java:225)
Error:         ch.ehi.ili2db.AbstractMain.domain(AbstractMain.java:554)
Error:         ch.ehi.ili2pg.PgMain.main(PgMain.java:70)
```

Here is a short example to reproduce the problem
`MULTISURFACE(CURVEPOLYGON(CIRCULARSTRING(0 0, 4 0, 4 4, 0 4, 0 0),(1 1, 3 3, 3 1, 1 1)))`

**Comment by signedav** [comment]
I'll have a look at it.

**Comment by claeis** [comment]
fixed in ili2db-4.4.6
